src/test/java/testresults/GetTestResults.java

```diff
@@ -41,6 +42,18 @@
     private TestResultsGet vehicleSubmittedData = TestResultsData.buildTestResultsSubmittedDataWithCalculated().build();
     private TestResultsGet vehicleCancelledData = TestResultsData.buildTestResultsCancelleddDataWithCalculated().build();
 
+    private String test_results_roadworthiness_hgv_pass_7675_json;
```

Comment:
vehicleTechnicalRecordsSteps on line 38 is greyed out, is that required now?

Reply:
done